### ISSUE TITLE
Export and Import 1000 PIDs

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#553]
For this ticket, we will run the export and import process on a set of 1000 pids.

Please send the CSV to Emily so she can spot-check it before we load it.
The works should be deposited into the Default Admin Set (they are not under workflow review).
The works should be deposited into the OpenEmory collection.

Please see the following [worksheet identifying the 1000 pids for this batch](https://docs.google.com/spreadsheets/d/1L4fNt6XOmkLj0o0hyCxwTuNmtquh_woJK-ojVWpetWw/edit?gid=1867210313#gid=1867210313).

To confirm: if any of these PIDs have already been exported to EFS, will this cause issues?

**Comment by bwatson78** [comment]
@eporter23 `if any of these PIDs have already been exported to EFS, will this cause issues?` A: yes, this will leave the Work in place, but create duplicates of the FileSets.

Because of this, I will remove the already ingested PIDs and run that list.

**Comment by bwatson78** [comment]
@eporter23 PID tvs6r returned an empty XML file. Double checking whether I get the same on my local machine.

**Comment by bwatson78** [comment]
It does. Removing that PID and rerunning the extraction script.

**Comment by bwatson78** [comment]
[pids_with_no_binaries_20241009T1652.txt](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNU0yQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1f19f18f2776bab71e03d32467d3fdd1bb65ecde/pids_with_no_binaries_20241009T1652.txt)

**Comment by bwatson78** [comment]
Migration scripts ran. Documents have been shared via Slack.

**Comment by eporter23** [comment]
@bwatson78 was `tvs6r` the only one that threw that no path in db registry error?

**Comment by bwatson78** [comment]
@eporter23 Yes, that was the only one of the bunch.

**Comment by eporter23** [comment]
Looking at the various counts generated, I am finding
224 PIDs had no "content" data streams (which I've adjusted in my SOLR query)
1 PID had no db path in registry error (aka not in Fedora)
772 PIDs were processed for the Bulkrax CSV input
There were 1000 PIDs in the original set, so somehow we are missing 3 PIDs.

**Comment by bwatson78** [comment]
Doing a diff on the list as well as the intake docs.

**Comment by bwatson78** [comment]
These weren't a part of the original intake, for some reason. Analyzing them now.

-cwkt7
-cwkvc
-cwmr2

**Comment by bwatson78** [comment]
They all would qualify, having binaries. I really don't know why I left those off of the list. @eporter23 Would you like the separate ingest CSV for these?

**Comment by eporter23** [comment]
Since my original PID list contained unwanted IDs due to my having a bad SOLR query, no worries. Next time we do this I'll factor in deleting what's in EFS and starting fresh because I think it will get really complicated trying to figure out what is where.

**Comment by eporter23** [comment]
Technically the import is still running so I am keeping this open to discuss next steps for how to keep a handful of straggler jobs from allowing us to keep track of end times.

**Comment by eporter23** [comment]
Following our breakout session on 10/18, I've attempted some manual deletes and retries on the jobs that Bulkrax reported errors on, using the Bulkrax retry option (remove and rebuild). The CreateRelationshipsJobs continued to loop and not complete, and I eventually deleted them.  Of those still showing up as failures in the Bulkrax importer (6 out of 772), the works were all created, but 5/6 have not received their Collection relationship assignment. Additionally, one work (v1tp5) that was missing its FileSets triggered a` RunTimeError:Version unable to be created error`, though its FileSets do seem to have attached after a manual retry through Bulkrax.
